src/screens/Topics/index.js

```diff
@@ -87,6 +90,12 @@ const Topics = () => {
       Analytics.logEvent('onb_select_topics_add_btn', {
         onb_topics_selected: topicSelected
       });
+      AnalyticsEventTracking.eventTrack(
+        BetterSocialEventTracking.ONBOARDING_TOPICS_TOTAL_FOLLOWING,
+        {
+          total: topicSelected?.length
+        }
+      );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of event tracking using `AnalyticsEventTracking.eventTrack` seems fine. However, please ensure that `BetterSocialEventTracking.ONBOARDING_TOPICS_TOTAL_FOLLOWING` and `topicSelected?.length` are always defined to avoid sending undefined values to your analytics service.
```diff
+ AnalyticsEventTracking.eventTrack(
+   BetterSocialEventTracking.ONBOARDING_TOPICS_TOTAL_FOLLOWING,
+   {
+     total: topicSelected?.length
+   }
+ );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->